orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy

```diff
@@ -91,13 +91,6 @@ class TaskTasklet implements Tasklet {
 
         logResult(result, stage, chunkContext)
 
-        // we should reload the execution now, in case it has been affected
-        // by a parallel process
-        long scheduledTime = stage.scheduledTime
```

Comment:
Yeah, there's a whole bunch of similar things that existed prior to when we remodeled the way the saves happened. That line in particular does look like cruft that probably had a bunch more code around it at one time.